@nav-frontend/react-accordion/src/Accordion.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { forwardRef, useEffect, useRef, useState } from "react";
+import cls from "classnames";
+import {
+  UnmountClosed,
+  Collapse,
+  CollapseProps,
+  CollapseCallbackArgs,
+} from "react-collapse";
+
+import { Expand } from "@nav-frontend/icons";
+import "@nav-frontend/accordion-styles";
+import { guid } from "nav-frontend-js-utils";
+
+interface AccordionProps {
+  children: React.ReactNode;
+  title: React.ReactNode;
+  open?: boolean;
+  onClick?: (event: React.SyntheticEvent<HTMLButtonElement>) => void;
+  className?: string;
+  renderContentWhenClosed?: boolean;
+  collapseProps?: Partial<CollapseProps>;
+  buttonId?: string;
+}
+
+const Accordion = forwardRef<HTMLDivElement, AccordionProps>(
+  (
+    {
+      children,
+      title,
+      open = false,
+      collapseProps,
+      className,
+      renderContentWhenClosed = false,
+      onClick,
+      buttonId = guid(),
+      ...rest
+    },
+    ref
+  ) => {
+    const contentId = useRef(guid());
+    const [internalOpen, setInternalOpen] = useState<boolean>(open);
+
+    useEffect(() => {
+      setInternalOpen(open);
+    }, [open]);
+
+    const CollapseComponent = renderContentWhenClosed
+      ? Collapse
+      : UnmountClosed;
+
+    const handleClick = (e) => {
+      e.preventDefault();
+      if (onClick) {
+        onClick(e);
+      } else {
+        setInternalOpen(!internalOpen);
+      }
+    };
+
+    const onRest = (args: CollapseCallbackArgs) => {
+      if (collapseProps && collapseProps.onRest) {
```

Comment:
`?.`

Reply:
Var lagt til som en placeholder for draft.
https://github.com/navikt/nav-frontend-moduler/blob/bfc031c5e9aa13525053f684d1ede3497c0d5981/packages/nav-frontend-ekspanderbartpanel/src/ekspanderbartpanel-base.tsx#L77

I løsningen idag blir en intern onRest brukt for å hindre at man kan tabbe ut av knappen når panelet er i lukkeanimasjonen: https://github.com/navikt/nav-frontend-moduler/blob/bfc031c5e9aa13525053f684d1ede3497c0d5981/packages/nav-frontend-ekspanderbartpanel/src/ekspanderbartpanel-base.tsx#L86

Eneste grunn for dette er kanskje for å være litt mer brukervennlig da det kan være litt forvirrende at fokus endrer seg når det foregår store animasjoner?

Reply:
Så jeg vil fjernet den, men tenker det måre være en grunn for at den var lagt til?

Reply:
Gikk på skattejakt og fant denne fra tidenes morgen: https://github.com/navikt/nav-frontend-moduler/commit/693026231d3aba89857a0458792390bf66dc4a0f
Så ser egentlig ikke noe grunn for å beholde denne interaksjonen